packages/notification-email-sender/src/services/notificationEmailSenderService.ts

```diff
@@ -35,20 +38,19 @@ import {
 import { ReadModelService } from "./readModelService.js";
 
 // Be careful to change this enum, it's used to find the html template files
-export const agreementEventMailTemplateType = {
+export const eventMailTemplateType = {
   activation: "activation-mail",
   submission: "submission-mail",
   rejection: "rejection-mail",
+  aboveTheThreshold: " estimate_above_the_threshold",
```

Comment:
```suggestion
  aboveThreshold: " estimate-above-threshold",
```
Let's keep the same naming with `-` instead of `_`